priiloader/hacks_hash.ini

```diff
@@ -548,3 +548,15 @@ minversion=1
 amount=1
 hash=0x2c030000,0x418200ac,0x38000001,0x981f0010
 patch=0x480000b0
+[No-Block RAAE,408x,410x]
+maxversion=513
+minversion=417
+amount=1
+hash=0x28009ea3,0x4082009c
+patch=0x28009ea3,0x4082009c,0x4280006c
+[Remove IOS16 Backup Disc Check]
+maxversion=513
+minversion=417
+amount=1
+hash=0x68630010
+patch=0x68630010,0x7c600379,0x4280001c
```

Comment:
isn't it easier to just patch it to an ios thats just not legit like 0xFF or 0x00 , instead of fucking with the xori instruction

Reply:
This patch doesn’t actually touch the xori instruction; it’s just there for the hash. The patch itself changes the `bne` afterward to a `b`.

Reply:
thats my point, isn't it better to make the code compare it to an impossible value instead of branching somewhere? 
doesn't that kinda do the same?

Reply:
I think it's cleaner to either have this unconditional branch or move the unconditional branch a few instructions up to skip the unnecessary code to load the IOS16 ID into registers.

Reply:
so, i just reverse engineered the code myself a bit, and i think if you replace the 0x68630010 with 0x68630000 it will xor with 0, which will never return 0 ( unless you use IOS0 ) and therefor not do whats in the if.
on top of that, if the tmd contains a titleID that is not IOS, it will still fail.

aka, much better than your patch.

Reply:
Doesn't that leave an opening for issues if a theoretical disc using 1-0 (for whatever reason) is discovered though? On the other hand, replacing the LWZ's with LI's keeps the same comparison logic there, while removing this potential issue (like with how your AHBPROT thing works), and the system version is read a second time later when it actually needs to reload.

Reply:
what is 1-0? lowest IOS I know of is IOS3

Reply:
That's the point of setting it to 1-0. I patched the disc IOS to something else though because I felt like it.

Reply:
1-0 is an invalid System Title. 1-1 is boot2, 1-2 is system menu. 
if anything i'd assume that if 1-0 exists, its boot1, seeing the system title numbers logic.
i still believe my proposal of the patch is the better patch, as it keeps all checks incheck, but just invalidates on part of the if statement so it looks like this 
![image](https://user-images.githubusercontent.com/12701306/175863665-5a425ff0-5dce-49e0-8a00-307ba9dea7ee.png)

Reply:
New version has been pushed.